scoap3/modules/workflows/workflows/articles_upload.py

```diff
@@ -53,6 +54,25 @@ def __get_first_doi(obj):
     return obj.data['dois'][0]['value']
 
 
+def __halt_and_notify(msg, obj, eng):
+    eng.halt(msg)
+
+    ctx = {
+        'doi': __get_first_doi(obj),
+        'reason': msg,
+        'url': url_for('workflow.details_view', id=eng.current_object.workflow.uuid, _external=True)
+    }
+
+    msg = TemplatedMessage(
+        template_html='scoap3_workflows/emails/halted_article_upload.html',
+        subject='SCOAP3 - Artcile upload',
```

Comment:
SInce this is a halt email, maybe it would be better to have a bit different title: 'SCOAP3 - Artcile upload halted'?